store/src/state.rs

```diff
@@ -420,7 +421,11 @@ impl State {
 
         let account = AccountState {
             account_id,
-            account_hash: inner.account_tree.get_leaf(account_id)?,
+            account_hash: inner
+                .account_tree
+                .open(&LeafIndex::new_max_depth(account_id))
```

Comment:
Same comment as above.

Reply:
This one too - this is a symptom of the issue that we redefine `type AccountId = u64` in the store instead of using `AccountId` in `miden-base`, as mentioned [here](https://github.com/0xPolygonMiden/miden-node/issues/74#issuecomment-1894304657)